ctf/check.cmd

```diff
@@ -0,0 +1,12 @@
+@echo off
+%LUA_CPATH%\lua5.1 -v || goto :exit
+
+cd tests
+REM Each suite should run individually
+REM Add them here
+@echo on
+%LUA_CPATH%\lua5.1 core_suite.lua
+@echo off
+cd ..
+@echo on
+:exit
```

Comment:
If anyone reading this: don't define %LUA_CPATH% in your system environment. lua defines it with good default values.
It is better to add directory of lua5.1.exe to PATH.
I probably will not change this script because whole directory structure should be revised with "busted" framework.